lib/galaxy/tools/deps/container_resolvers/mulled.py

```diff
@@ -33,9 +33,9 @@
 
 
 def list_cached_mulled_images(namespace=None):
-    command = build_docker_images_command(truncate=True, sudo_docker=False)
-    command = "%s | tail -n +2 | tr -s ' ' | cut -d' ' -f1,2" % command
-    images_and_versions = check_output(command)
+    command = build_docker_images_command(truncate=True, sudo=False)
```

Comment:
This is also backporting commit 9f4add70d0967ccb6ddf1bd2d1393685ef639e97 from release 17.01, which fixes a nasty bug 👍